proposals/rook.adoc

```diff
@@ -0,0 +1,155 @@
+
+*Infrastructure requests (CI / CNCF Cluster):*
+
+CI currently at https://jenkins.rook.io but could move to CNCF CI.
+
+Planning to use CNCF cluster for integration and performance testing at scale.
+
+*Communication Channels:*
+
+* Slack: https://rook-slackin.herokuapp.com
+* Gitter: https://gitter.im/rook/rook (deprecated)
+* Google Groups: https://groups.google.com/forum/#!forum/rook-dev
+* Email: mailto:info@rook.io[info@rook.io]
+
+*Issue tracker:* https://github.com/rook/rook/issues
+
+*Website:* https://rook.io
+
+*Release methodology and mechanics:*
+
+Major releases roughly every two months, minor releases as needed.
+
+*Social media accounts:*
+
+* Twitter: @rook_io
+
+*Existing sponsorship*: Quantum Corp.
+
+In 2016 as part of ongoing product development work we identified the need for richer implementations of storage technologies in Cloud Native systems. As this work progressed we felt that it was evolving into a core component of the platform architecture and chose to open-source our work.
+
+Since then Quantum has continued to invest in both the Rook technologies and launching Rook as a vibrant open source project. Internally we are utilizing Rook as well as many other Cloud Native technologies to build systems relevant to our businesses.
+
+We firmly believe that a vibrant Rook project and ecosystem is in our and the community’s best interests. As the project continues to grow our role will become less significant in terms of strategy and direction and we think this evolution and adopting well established governance principles will strengthen the project.
+
+*Community size:*
+
+* Rook was open sourced Nov'2016
+* 1700+ stars
+* 34 contributors
+* 138 forks
+* 116 users on gitter, 80 on slack
+* 500K+ container pulls (quay.io), 50K+ container pulls (docker)
+
+*Comparison with gluster-kubernetes and ceph-container*:
+
+Existing approaches to running distributed storage systems like Ceph and Gluster focus primarily on packaging in containers, initial deployment, and bootstrapping. There is no central controller that is responsible for ongoing operations, dynamic management and maintenance of such storage systems. While some of these operations can be handled by the orchestration platform itself (for example, scaling through stateful-sets in Kubernetes) the approach only covers a small subset of the administration tasks and does not take into account the inherent constraints and guarantees of the backend storage system. For example, growing a cluster in Ceph not only requires scheduling more storage nodes but also updating the storage topology to optimize data access and improve durability all without breaking consistency guarantees. Rook's storage controller is responsible for ongoing and dynamic management of the storage system and it does so in a storage backend specific way.
```

Comment:
Eventhough I see the title as `comparison with gluster-kubernetes` , I failed to interpret the comparison in this section,  its really vague. Can you clarify what you mean by this section or make it clear ?